packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -256,3 +256,9 @@ enum GuardClassKind {
     }
   }
 }
+
+/// An extension on [Script] which allows for access to VM internal fields.
+extension ScriptPrivateViewExtension on Script {
+  static const _loadTimeKey = '_loadTime';
+  int? get loadTime => json![_loadTimeKey];
```

Comment:
This should never be null based on the implementation in the VM. We should be safe to add a null assertion here.